bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/p2/engine/ProvisioningContext.java

```diff
@@ -107,7 +107,17 @@ public boolean contains(IArtifactRepository element) {
 	 */
 	public static final String CHECK_AUTHORITIES = "org.eclipse.equinox.p2.director.checkAuthorities"; //$NON-NLS-1$
 
-	private static final String FOLLOW_ARTIFACT_REPOSITORY_REFERENCES = "org.eclipse.equinox.p2.director.followArtifactRepositoryReferences"; //$NON-NLS-1$
+	/**
+	 * Instructs the provisioning context to follow artifact repository references
+	 * when providing queryables for obtaining metadata and artifacts. When this
```

Comment:
I think this only affects artifact (and not metadata) repositories?

Reply:
You did tell him to reuse the documentation of the other constant which is documented in this same way.  😱

Reply:
Reading it again, it is not really "wrong" just a bit confusing, essentially it says then when "when providing queryables for obtaining metadata and artifacts" is created this influences only the former (or later in this case).